api/openapi-spec/swagger.json

```diff
@@ -15254,89 +15234,6 @@
       },
       "type": "object"
     },
-    "io.k8s.api.storage.v1alpha1.CSIStorageCapacity": {
```

Comment:
this doesn't belong in this PR... just realized why this kicked in. beta.0 was just tagged and this resource is marked to stop being served in 1.24, and that automatically turns off at beta

I'll make a separate commit you can rebase on that just has this change

Reply:
https://github.com/kubernetes/kubernetes/pull/109195 is open that just regenerates openapi on master

Reply:
I just did that, let's begin integration roulette once again.